### ISSUE TITLE
Public access users cant join conference

### ISSUE THREAD

**Issue by noriakisana** [OpenSlides/OpenSlides#6856]
**Describe the bug**
Users which access openslides via Public Access cant connect to the conference room.

**How to Reproduce**

1. Create an event
2. Add Conference data
3. Login via "public access"
4. try to connect to conference room
5. See error

**Expected behavior**
user could connect to conference room

**System information**

- OpenSlides version: 4.2

**Screenshots**
![Untitled](https://github.com/user-attachments/assets/3377a129-13ac-4e32-b917-1483c33a0d98)

**Comment by Elblinator** [comment]
The bug you are describing needs some conceptualising before anything can be implemented and depending on the concept this could be a new feature. We will do that internally in the future.

If you want your public access users to have access to your conference room, then a workaround would be to paste the conference url onto a a motion/the homepage/ or another page where you can paste text to and direct them to this url if they want to join

Can you please elaborate what your usecase is? Does your organisation want the public access users to be able to access the conference rooms? We expected too high traffic which in the worst case would lead to the conference room crashing

**Comment by noriakisana** [comment]
Yeah, so the workaround with the  linking on the start page is a currently our way to go.

Background of this usage is that we currently only create accounts for persons who are deligated to vote, others can join and follow the motions / elections via the public account.

In order to have a better user experience i thought it would be nice to have the ability as a public user (aka. guest) to join the conference without any other tabs open.

Currently we are working with a max of 70 people, whether max. 30 are online so high traffic for jitsi shall not be our problem currently, but i understand the concerns.

For the usernames which are ported to jitsi you could add a name field which pops up when you try to enter the conference or just a small random string.
